### DISCUSSION TITLE
Compile and setup problems

### DISCUSSION THREAD

**Discussion by ChrisLetti** [OwlGamingCommunity/V#5] [Support]
I cannot compile the project. I brought the project into Visual Studio 2019 via Git Clone and adjusted the database. When I try to compile the project, the console shows that the .NET SDK could not be found. This message comes for every sub-project.

Furthermore, I don't get along with the instructions for the setup. I don't understand what is meant by what is written by "Obtaining RAGE".

I would be very thankful for help.

I would like to use this project as a test server and as a programming aid.

Sorry for this English. I am German and use google translate.

**Comment by Test83318** [comment]
Copy the files from the (RAGEMP client (rage.mp) folder)/server_files into V/Output/Debug (so V/Output/Debug/ragemp_server.exe or the linux equivalent). Remember to enable the csharp option on the config.json.
Then be sure to install the window SDK, I recommend doing a manual installation. https://docs.microsoft.com/es-es/dotnet/core/install/windows?tabs=net50#download-and-manually-install
Finally you can right click the Boot Project and Set as start project or something like that; you press F5 and it should start. (99% chance of having issues with the database, but that's a beggining)

**Reply by DraigNET** [reply]
I've managed to get the server running but it doesn't load the game mode at all.

![image](https://user-images.githubusercontent.com/66184380/129414990-bd65cd99-c696-4787-b837-ece64f1cdd74.png)

**Comment by Test83318** [comment]
On the conf.json change:

    "csharp" : "disabled",
into:
    "csharp" : "enabled",

**Comment by ChrisLetti** [comment]
@ Test83318
I've done that so far. Now, when starting the server, I have the problem that it cannot find a file, but it is available at the path shown.

![owl_script_loader](https://user-images.githubusercontent.com/2155782/129441926-c9842b43-8993-407a-9d1e-bc9684e0af9f.png)

**Reply by Test83318** [reply]
Try to run the Cooker at Source/Cooker/bin/Debug/netcoreapp3.1 (or similar)/Cooker.exe

**Reply by ChrisLetti** [reply]
Thank you for your patience.

Always a new mistake. It's no longer fun.
The new error is port 3307. Target computer refuses connection 127.0.0.1:3307

**Reply by Test83318** [reply]
There are multiple ways to implement the database, the simplest is to change in the mysql project to hard code the database connection data. 

(It must be done at both
Source\owl_mysql\ThreadedMySQL.cs => Lines starting at 640
Source\owl_mysql\Mysql.cs => Lines starting at 63
 )

Remember that the mysql port by default is 3306

Don't worry about me, all these problems I have already had them when I tried to make it run. XD

**Reply by DraigNET** [reply]
So change AUTH_DATABASE_PORT to 3306?

System.Net.HttpListenerException: 'Access is denied.'

That's what I'm getting at the moment :\

_**Access is denied.
Access is denied.
Loaded map Church.ymap with 319 objects
Access is denied.
Loaded map ComedyClubFix.ymap with 28 objects
Access is denied.
Loaded map ComplexLobby.ymap with 68 objects
Loaded map DavisFireHelipad.ymap with 49 objects
Access is denied.**_

And that in the console.
